### DISCUSSION TITLE
Can it possibe to display sharepoint profile custome fields through pnp-modern-search webpart?If yes, please mention how?

### DISCUSSION THREAD

**Discussion by swapnapaul141** [microsoft-search/pnp-modern-search#1726] [Q&A]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by wobba** [comment]
If your custom user profile property is exposed as a crawled property and mapped to a managed property, you can can retrieve it via the SharePoint search provider. I touch on this a bit in my old post https://www.techmikael.com/2014/11/how-to-make-sure-user-profile.html
